python/mxnet/gluon/block.py

```diff
@@ -502,8 +502,16 @@ def hybridize(self, active=True, **kwargs):
         ----------
         active : bool, default True
             Whether to turn hybrid on or off.
-        **kwargs : string
-            Additional flags for hybridized operator.
+        static_alloc : bool, default False
+            Statically allocate memory to improve speed. Memory usage may increase.
+        static_shape : bool, default False
+            Optimize for invariant input shapes between iterations. Must also
+            set static_alloc to True. Change of input shapes is still allowed
```

Comment:
can we assert that static_alloc is set if static_shape is set so that we don't open the door to undefined behaviours?
```
if static_shape:
    assert static_alloc, "static_alloc must be True if static_shape is True"
```